src/charts/Metrics.tsx

```diff
@@ -27,46 +27,50 @@ export const colors = {
   secondary: '#7f8c8d',
 };
 
-const IconWrapper = styled.div`
-  display: flex;
-  align-items: center;
-  padding-left: 0.2rem;
-  flex-grow: 1;
-`;
+export const IconWrapper = ({ id, children }: { id: string; children: React.ReactNode }) => {
```

Comment:
The way you've done these wrappers is not ideal, because they don't support arbitrary props that the base element (`div`) supported (e.g. `onClick` with the correct type). You could write your own correct wrapper, but I wouldn't do that. There's already [a library to make these wrappers](https://github.com/MathiasGilson/tailwind-styled-component). Of course for places where you're using `className` directly without defining a reusable wrapper component, you should keep doing that.

Reply:
It might be best to open an issue for this and fixing it immediately _after_ this PR is merged.